.circleci/config.yml

```diff
@@ -67,11 +67,23 @@ commands:
             - /home/circleci/.composer/cache/
       - store_artifacts:
           path: ./artifacts
-      - slack/status:
-          fail_only: true
-          failure_message: ':red_circle: M2: A $CIRCLE_JOB job has failed!'
-          only_for_branches: master
-          webhook: $SLACK_GREENKEEPER_WEBHOOK
+      - run:
+          name: "Setup branch string"
+          when: on_fail
+          command: |
+            export FILTER_BRANCHES=$(<./.circleci/scripts/auto-rebase-branches.txt)
+            if [[ "$FILTER_BRANCHES" == *"$CIRCLE_BRANCH"* ]]; then
```

Comment:
What syntax is this? It looks like a regular expression matching, but can you a reference? 

I think we want the exact match rather than matching pattern. For example, any branch name that contain "yogobo"( jz/yogobo, ci/yogibo, yogibo-2.0.6 ) would notify when yogibo is on the list

Reply:
Actually  when yogibo is on the list only ci/yogibo should notify as that is the one that gets the rebase push.
https://github.com/BoltApp/bolt-magento2/blob/master/.circleci/scripts/auto-rebase.sh#L36

Reply:
Something like:

` [[ ${CIRCLE_BRANCH:0:3} == ci/ ]] && grep -n "^${CIRCLE_BRANCH:3}$" ./auto-rebase-branches.txt`